service/src/main/java/org/broadinstitute/listener/relay/inspectors/SetDateAccessedInspector.java

```diff
@@ -152,4 +150,22 @@ private synchronized boolean hasLastAccessDateExpired() {
   private synchronized void updateLastAccessedDate() {
     lastAccessedDate = lastAccessedDate.plusSeconds(callWindowInSeconds);
   }
+
+  private String httpRequestToString(HttpRequest request) {
```

Comment:
Doesn't seem like these methods are used?

Reply:
Is there a local file missing from this PR that would make use of these new methods? If not I would remove to avoid confusion

Reply:
Ha! Purely here for debug purposes. Thanks